src/pages/instance-mapping/InstanceMapping.tsx

```diff
@@ -198,6 +199,50 @@ const InstanceMappingPage: React.FC = () => {
         applyMapping(selection, undefined);
     };
 
+    const applyAutoMapping = async (selection: string[]) => {
+        const selectedItem = _.find(rows, ["id", selection[0]]);
+
+        if (!instance) {
+            snackbar.error(i18n.t("Please select an instance from the dropdown"), {
+                autoHideDuration: 2500,
+            });
+            return;
+        } else if (selection.length !== 1) {
+            snackbar.error(i18n.t("Auto-mapping does not support multiple action yet"), {
+                autoHideDuration: 2500,
+            });
+            return;
+        } else if (!selectedItem) {
+            snackbar.error(i18n.t("Unexpected error, could not apply auto mapping"), {
+                autoHideDuration: 2500,
+            });
+            return;
+        }
+
+        const { objects: candidates } = await instance
+            .getApi()
+            //@ts-ignore
+            .models[type].get({
+                fields: { id: true },
+                filter: {
+                    name: { ilike: selectedItem.name },
```

Comment:
@SferaDev Not sure about the logic here. selectedItem.name is empty. Maybe we can use displayName o modify the model. Apart from that, I have some doubts regarding the API call. For example, create a new data element with the same name than in the origin instance and a completely different short name and code. The auto mapping feature cannot find the data elements. I guess the API call needs to be refined...